app/views/catalog_admin/fields/_embed_option_inputs.html.erb

```diff
@@ -0,0 +1,23 @@
+<div data-controller="toggle-display" data-toggle-display-value='url'>
+  <%= f.collection_select(:format, Field::Embed::FORMATS, :to_s, :to_s, {}, {
+    "data-action": "input->toggle-display#reveal",
+    "data-toggle-display-target": "source",
+  }) %>
+
+  <div data-toggle-display-target="revealable">
+    <%= f.number_field(:width, :label => t('.width')) %>
+    <%= f.number_field(:height, :label => t('.height')) %>
+  </div>
+
+  <%= f.hidden_field(:domains, required: true) %>
+  <div class="mt-2">
+    <%= react_component('Domains/components/Domains',
+                        {
+                          domains: f.object.parsed_domains,
+                          options: load_domains,
+                          inputId: "field_domains",
+                          placeholder: t(".domains_component_placeholder"),
+                          noOptionsMessage: t(".domains_component_no_options")
+                        }) %>
+  </div>
```

Comment:
![Capture d’écran 2021-11-09 à 19 43 52](https://user-images.githubusercontent.com/417818/140985504-779cd51b-604d-4908-a69a-b473b25d546e.png)

Reply:
put just above format (not to mess with revealable width/height)